docker-widget/docker.lua

```diff
@@ -19,9 +19,12 @@ local HOME_DIR = os.getenv("HOME")
 local WIDGET_DIR = HOME_DIR .. '/.config/awesome/awesome-wm-widgets/docker-widget'
 local ICONS_DIR = WIDGET_DIR .. '/icons/'
 
-local LIST_CONTAINERS_CMD = [[bash -c "docker container ls -a -s -n %s]]
+local LIST_CONTAINERS_CMD = [[bash -c "%s container ls -a -s -n %s]]
     .. [[ --format '{{.Names}}::{{.ID}}::{{.Image}}::{{.Status}}::{{.Size}}'"]]
 
+local DOCKER_DEFAULT_STATUS_PATTERN = '(.*)::(.*)::(.*)::(%w*) (.*)::(.*)'
+local DOCKER_CREATED_STATUS_PATTERN = '(.*)::(.*)::(.*)::Created::(.*)'
```

Comment:
I've added a new pattern to match if the container status is `Created`. Here are two examples for reference:
```
testcontainer1::8ab38957029c::localhost/db-transfer-postgres:latest::Up 6 seconds::63B (virtual 38.2MB)
testcontainer2::008107c11e2d::mcr.microsoft.com/mssql/server:2022-latest::Created::289MB (virtual 1.6GB)
```
Since `Created` containers have never started they do not have a time component.